lib/bitcoin/MongoDbServiceStateStore.ts

```diff
@@ -0,0 +1,32 @@
+import IServiceStateStore from './interfaces/IServiceStateStore';
+import MongoDbStore from './MongoDbStore';
+
+/**
+ * Implementation of IServiceStateStore using MongoDB database.
+ */
+export default class MongoDbServiceStateStore<T> extends MongoDbStore implements IServiceStateStore<T> {
```

Comment:
Is this templatized for future use? As in other services may also use this store for their own service version? If yes, then does this belong into the `common` folder?

Reply:
This is definitely written in preparation for future service usage. But I left it in bitcoin because "right now" there isn't other usage. But no push back to move it right now.